doc/admin/admin_commands/kdb5_util.rst

```diff
@@ -90,13 +96,18 @@ create
 
 .. _kdb5_util_create:
 
-    **create** [**-s**]
+    **create** [**-s**] [**-h**]
```

Comment:
Should we mention `-W` here?

Also, I don't think it's actually possible to engage the `-h` behavior right now; it's not in the `getopt()` call: https://github.com/krb5/krb5/blob/master/src/kadmin/dbutil/kdb5_create.c#L167

Reply:
Interesting.  Commit 21ddb0df0b5394279f82480257c0cf32f45810a1 attempted to add kdb5_util create -h, but in the intervening 21 years no one has ever noticed that it never made it into the getopt string.  I will change the PR to remove the switch case for the option instead of documenting it.

(This also suggests that we could get rid of kdb5_util load -hash without bothering anyone, but I won't go that far.)